### PR TITLE
docs: update log

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | docs: update ChangeLog |
| 🇨🇳 Chinese |         -  |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 560f6ef</samp>

This pull request updates the English and Chinese changelogs for version 5.5.0 of ant-design with a bug fix entry. The bug fix removes the border around the tags in multiple selection components.

### <samp>🤖 Generated by Copilot at 560f6ef</samp>

* Add a bug fix entry to the English and Chinese changelogs for version 5.5.0 ([link](https://github.com/ant-design/ant-design/pull/42389/files?diff=unified&w=0#diff-e961ad288c43a2feb2d56b696e10c0b670d652dbb17c76c9aa0452d45157977cR36), [link](https://github.com/ant-design/ant-design/pull/42389/files?diff=unified&w=0#diff-1224d5ca5233fdeba67ba2e2dc6fa3e615905253f35dc132c4b4b86f51f9eab4R36))

### CODE REVIEW

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ timeline: true
   - 🐞 修复 ConfigProvider `size` 对 Steps 无效的问题。[#42278](https://github.com/ant-design/ant-design/pull/42278) [@wanghui2021](https://github.com/wanghui2021)
   - 🐞 修复 ConfigProvider `size` 对 Descriptions 无效的问题。[#42244](https://github.com/ant-design/ant-design/pull/42244) [@wanghui2021](https://github.com/wanghui2021)
   - 🐞 修复当 ConfigProvider 中 `componentSize` 被设定时, Space.Compact 没有继承的问题。[#42199](https://github.com/ant-design/ant-design/pull/42199) [@Ec-tracker](https://github.com/Ec-tracker)
+- 🐞 优化 `Select`、`Tree-select`、`Cascader` 多选模式下的样式，去除标签的边框。[#41480](https://github.com/ant-design/ant-design/pull/41480)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:36]:
```suggestion
- 💄 优化 `Select`、`Tree-select`、`Cascader` 多选模式下的样式，去除标签的边框。[#41480](https://github.com/ant-design/ant-design/pull/41480)
```

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ timeline: true
   - 🐞 Fix ConfigProvider `size` prop not work on Steps. [#42278](https://github.com/ant-design/ant-design/pull/42278) [@wanghui2021](https://github.com/wanghui2021)
   - 🐞 Fix ConfigProvider `size` prop not work on Descriptions. [#42244](https://github.com/ant-design/ant-design/pull/42244) [@wanghui2021](https://github.com/wanghui2021)
   - 🐞 Fix the size is not inherited from ConfigProvider when the `componentSize` is set. [#42199](https://github.com/ant-design/ant-design/pull/42199) [@Ec-tracker](https://github.com/Ec-tracker)
+- 🐞 Optimize the style of `Select`, `Tree-select`, `Cascader` multi-selection mode, remove the Tag border. [#41480](https://github.com/ant-design/ant-design/pull/41480)
```
Comment (li-jia-nan) [inline CHANGELOG.en-US.md:36]:
```suggestion
- 💄 Optimize the style of `Select`, `Tree-select`, `Cascader` multi-selection mode, remove the Tag border. [#41480](https://github.com/ant-design/ant-design/pull/41480)
```

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -33,6 +33,7 @@ timeline: true
   - 🐞 修复 ConfigProvider `size` 对 Steps 无效的问题。[#42278](https://github.com/ant-design/ant-design/pull/42278) [@wanghui2021](https://github.com/wanghui2021)
   - 🐞 修复 ConfigProvider `size` 对 Descriptions 无效的问题。[#42244](https://github.com/ant-design/ant-design/pull/42244) [@wanghui2021](https://github.com/wanghui2021)
   - 🐞 修复当 ConfigProvider 中 `componentSize` 被设定时, Space.Compact 没有继承的问题。[#42199](https://github.com/ant-design/ant-design/pull/42199) [@Ec-tracker](https://github.com/Ec-tracker)
+- 💄 优化 `Select`、`Tree-select`、`Cascader` 多选模式下的样式，去除标签的边框。[#41480](https://github.com/ant-design/ant-design/pull/41480)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:36]:
```suggestion
- 💄 优化 Select、Tree-select、Cascader 多选模式下的样式，去除标签的边框。[#41480](https://github.com/ant-design/ant-design/pull/41480)
```

Comment (afc163) [reply CHANGELOG.zh-CN.md:36]: 可以放前面去。

Comment (li-jia-nan) [reply CHANGELOG.zh-CN.md:36]: 放在新组件的下面吗？

Comment (afc163) [reply CHANGELOG.zh-CN.md:36]: 对。

Comment (li-jia-nan) [reply CHANGELOG.zh-CN.md]: done~

**Review by zombieJ — APPROVED** [review]

File: CHANGELOG.zh-CN.md (hunk 3 above)
Comment (li-jia-nan) [reply CHANGELOG.zh-CN.md]: 这个还有什么问题吗大佬，我已经把顺序调整到新组件的下面了

Comment (li-jia-nan) [reply CHANGELOG.zh-CN.md]: 反引号也去掉了

**Comment by afc163** [comment]
图片

**Comment by li-jia-nan** [comment]
哦哦，是要用大驼峰，我再改一下

**Comment by li-jia-nan** [comment]
> 哦哦，是要用大驼峰，我再改一下

- #42410

File: CHANGELOG.zh-CN.md (hunk 3 above)
Comment (li-jia-nan) [reply CHANGELOG.zh-CN.md]: - #42410

**Pull request merged**